docs/reference/spec.md

```diff
@@ -49,7 +49,7 @@ e.g `authorization/foo-org-xyz` references the Foo Organization.
 
 ## Inception
```

Comment:
@dberesford the specs doc has too much content to be put on the main readme. As a higher level specs doc it seems useful to me and complementary to the lower level technical explanation doc. I'll review it and see what can I do about it.